module/VuFind/src/VuFind/RecordDriver/ProQuestFSG.php

```diff
@@ -0,0 +1,97 @@
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  RecordDrivers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Maccabee Levine <msl321@lehigh.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:record_drivers Wiki
+ */
+
+namespace VuFind\RecordDriver;
+
+use VuFind\RecordDriver\Feature\MarcAdvancedTrait;
+use VuFind\RecordDriver\Feature\MarcBasicTrait;
+use VuFind\RecordDriver\Feature\MarcReaderTrait;
+
+/**
+ * Model for MARC records in ProQuest Federated Search Gateway.
+ *
+ * @category VuFind
+ * @package  RecordDrivers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Maccabee Levine <msl321@lehigh.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:record_drivers Wiki
+ */
+class ProQuestFSG extends DefaultRecord
+{
+    use MarcReaderTrait, MarcAdvancedTrait, MarcBasicTrait {
+        MarcBasicTrait::getNewerTitles insteadof MarcAdvancedTrait;
+        MarcBasicTrait::getPreviousTitles insteadof MarcAdvancedTrait;
+        MarcBasicTrait::getShortTitle as marcGetShortTitle;
+        MarcBasicTrait::getTitle as marcGetTitle;
+    }
+
+    /**
+     * Set raw data to initialize the object.
+     *
+     * @param mixed $data Raw data representing the record; Record Model
+     * objects are normally constructed by Record Driver objects using data
+     * passed in from a Search Results object. In this case, $data is a MARCXML
+     * document.
+     *
+     * @return void
+     */
+    public function setRawData($data)
+    {
+        // Ensure that $driver->setRawData($driver->getRawData()) doesn't blow up:
+        if (isset($data['fullrecord'])) {
+            $data = $data['fullrecord'];
+        }
+
+        // Map the ProQuestFSG response into a format that the parent Solr-based
+        // record driver can understand.
+        parent::setRawData(['fullrecord' => $data]);
+    }
+
+    /**
+     * Get the short (pre-subtitle) title of the record.
+     *
+     * @return string
+     */
+    public function getShortTitle()
+    {
+        return strip_tags($this->marcGetShortTitle());
+    }
+
+    /**
+     * Get the full title of the record.
+     *
+     * @return string
+     */
+    public function getTitle()
+    {
+        return strip_tags($this->marcGetTitle());
```

Comment:
This addresses @sturkel89 's comment

> HTML code appearing in titles in results browse

Although like `getShortTitle` I actually do want to keep HTML in there when it's useful, after #4004.

Reply:
> Ideally, the Latin term would be displayed in italics.

Yes exactly.  I'm waiting on #4004 for the infrastructure to do that well.